include/kernels/gauge_fix_fft.cuh

```diff
@@ -70,10 +70,10 @@ namespace quda {
     }
   };
 
-  template <typename store_t, QudaReconstructType recon>
+  template <typename Float_, QudaReconstructType recon>
```

Comment:
Why this change?

Reply:
I thought I fixed this back to store_t

Reply:
I will revert this.

Reply:
Reverted and fixed with 2bc5b001c. - there was one unset result in there that I initialized further down in the file.